test/integration.test.js

```diff
@@ -35,7 +35,7 @@ let env;
 
 // Environment variables used only for the local npm command used to do verification
 const npmTestEnv = {
-  ...process.env,
+  ...processEnvWithoutGitHubActionsVariables,
```

Comment:
not directly impactful now that i backed away from specifying the `testdouble` loader with `NODE_OPTIONS`, but realized that we were not doing this here while debugging that situation